files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/tools.yml

```diff
@@ -41,8 +41,12 @@ tools:
   toolshed.g2.bx.psu.edu/repos/nml/spades/spades/.*:
     cores: 2
     rules:
-      - if: 0.005 <= input_size < 2
+      - if: 0.005 <= input_size < 0.1
         cores: 8
+        mem: 16
+      - if: 0.1 <= input_size < 2
```

Comment:
This means that with 100MB input file we already need 200GB of memory?

Reply:
Yes, probably it would be better to increase until 1GB.